v2/e2e/Makefile

```diff
@@ -102,4 +102,4 @@ clean:
 
 .PHONY: setup
 setup:
-	go install github.com/onsi/ginkgo/ginkgo
+	go install github.com/onsi/ginkgo/ginkgo@v1.16.4
```

Comment:
Why don't we stop using `ginkgo`?
It is merely a wrapper around `go test`.

Reply:
We didn't use `ginkgo` in the first place.